src/buildtools/fslex/fslex.fs

```diff
@@ -173,52 +172,51 @@ let main(args: string[]) =
                   outputCodedUInt16 os sentinel
             for i = 0 to 255 do 
                 let c = char i
-                emit (EncodeChar c);
-            emit Eof;
+                emit (EncodeChar c)
+            emit Eof
             cfprintfn os "|];"
-        done;
+        done
     
-    cfprintfn os "    |] ";
+    cfprintfn os "    |] "
     
-    fprintf os "let actions : uint16[] = [|";
+    fprintf os "let actions : uint16[] = [|"
     for state in dfaNodes do
         if state.Accepted.Length > 0 then 
           outputCodedUInt16 os (snd state.Accepted.Head)
         else
           outputCodedUInt16 os sentinel
-    done;
-    cfprintfn os "|]";
-    cfprintfn os "let _fslex_tables = %s.%sTables.Create(trans,actions)" lexlib domain;
+    done
+    cfprintfn os "|]"
+    cfprintfn os "let _fslex_tables = %s.%sTables.Create(trans,actions)" lexlib domain
     
-    cfprintfn os "let rec _fslex_dummy () = _fslex_dummy() ";
+    cfprintfn os "let rec _fslex_dummy () = _fslex_dummy() "
 
     // These actions push the additional start state and come first, because they are then typically inlined into later
     // rules. This means more tailcalls are taken as direct branches, increasing efficiency and 
     // improving stack usage on platforms that do not take tailcalls.
     for ((startNode, actions),(ident,args,_)) in List.zip perRuleData spec.Rules do
-        cfprintfn os "(* Rule %s *)" ident;
-        cfprintfn os "and %s %s (lexbuf : %s.LexBuffer<_>) = _fslex_%s %s %d lexbuf" ident (String.Join(" ",Array.ofList args)) lexlib ident (String.Join(" ",Array.ofList args)) startNode.Id;
-    for ((startNode, actions),(ident,args,_)) in List.zip perRuleData spec.Rules do
-        cfprintfn os "(* Rule %s *)" ident;
-        cfprintfn os "and _fslex_%s %s _fslex_state lexbuf =" ident (String.Join(" ",Array.ofList args));
-        cfprintfn os "  match _fslex_tables.Interpret(_fslex_state,lexbuf) with" ;
-        actions |> Seq.iteri (fun i (code,pos) -> 
-            cfprintfn os "  | %d -> ( " i;
-            cfprintfn os "# %d \"%s\"" pos.Line pos.FileName;
+        cfprintfn os "// Rule %s" ident
```

Comment:
This must be the change where it fixes the issue.

Reply:
Yes that's right